newbank/client/ClientDisplay.java

```diff
@@ -14,6 +14,12 @@ public static void clientServiceIntroduction(){
         System.out.println("|-----------------------------------|");
     }
 
+    public static void clientCreatedAnnouncement(String password){
+        System.out.println("Congratulations, your account as been created !");
+        System.out.println("£10 has been credited on this account as a welcome gift.");
+        System.out.println("To connect later on, your password is: " + password);
+    }
```

Comment:
Currently this method is called and printed on the server and not the client, potentially it would be better to take a second argument that allows you to pass the output, for example:

    public static void clientCreatedAnnouncement(String password, PrintWriter out){
        out.println("Congratulations, your account as been created !");
        out.println("£10 has been credited on this account as a welcome gift.");
        out.println("To connect later on, your password is: " + password);
    }

Then this can be called from the server as:

clientDisplay.clientCreatedAnnouncement(password, out);

Reply:
Agreed, I had tried to return a string instead, but I think your solution is better!